pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/utils/SchemaValidationFunctions.kt

```diff
@@ -0,0 +1,374 @@
+package gov.cdc.ocio.processingstatusapi.utils
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.JsonNodeType
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.google.gson.ToNumberPolicy
+import com.networknt.schema.JsonSchema
+import com.networknt.schema.JsonSchemaFactory
+import com.networknt.schema.SpecVersion
+import com.networknt.schema.ValidationMessage
+import gov.cdc.ocio.processingstatusapi.ReportManager
+import gov.cdc.ocio.processingstatusapi.exceptions.BadRequestException
+import gov.cdc.ocio.processingstatusapi.models.reports.CreateReportSBMessage
+import gov.cdc.ocio.processingstatusapi.models.reports.MessageMetadata
+import gov.cdc.ocio.processingstatusapi.models.reports.Source
+import gov.cdc.ocio.processingstatusapi.models.reports.StageInfo
+import gov.cdc.ocio.processingstatusapi.utils.Helpers.logger
+import gov.cdc.ocio.processingstatusapi.utils.Helpers.reason
+
+import mu.KotlinLogging
+import java.awt.datatransfer.MimeTypeParseException
+import java.io.File
+import java.time.Instant
+import java.util.*
+import javax.activation.MimeType
+
+
+
+object Helpers {
```

Comment:
Let's encapsulate all of this in a class called SchemaValidation.  Make the contents of the Helpers a companion object of the SchemaValidation class.